src/main/resources/application.yaml

```diff
@@ -209,7 +209,7 @@ uk:
             APPLICANT1_SOLICITOR_APPLIED_FOR_CONDITIONAL_ORDER: 'ee403b64-1df6-4630-a233-881e85be0395'
             JOINT_APPLIED_FOR_CONDITIONAL_ORDER: '0d8c59f1-aea3-4245-8bca-eb0d5558f0fd'
             JOINT_PARTNER_APPLIED_FOR_CONDITIONAL_ORDER: '444fbd4b-5c40-4e42-99a0-761cd07933a6'
-            JOINT_BOTH_APPLIED_FOR_CONDITIONAL_ORDER: 'aa6114e5-f652-49ab-82f9-66d1b2deefed'    #TODO: Welsh translation needed
+            JOINT_BOTH_APPLIED_FOR_CONDITIONAL_ORDER: '6caef42a-91d7-40e5-b440-cbb48daf1415'    #TODO: Welsh translation needed
```

Comment:
can we please remove the comment ?

Reply:
The comment is still correct - there is no actual welsh translation for this template.